moped-editor/src/theme/index.js

```diff
@@ -23,6 +23,7 @@ const theme = createMuiTheme({
     },
     map: {
       transparent: "rgba(0,0,0,0)",
+      trail: "#4caf50",
```

Comment:
It shouldn't go without saying, I like the color and it makes it easy to the eye and it makes the app look more Kawaii